data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes lopen over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
In het geval van 'achtbaanrails' en ook bijvoorbeeld 'treinrails', heeft 1 spoor altijd 2 rails die samen 1 spoor maken, derhalve is een achtbaanrails gewoon correct. 'een treinspoor = een treinrails'

Zie ook 'enkele rail' en 'monorail'

Valt me trouwens nu ineens op, maar ''Bootvormige karretjes lopen'' heb ik gelijk maar veranderd in ''Bootvormige karretjes rijden''.